detekt/defs.bzl

```diff
@@ -79,9 +81,10 @@ detekt = rule(
             default = "@detekt_cli_jar//file",
             allow_single_file = True,
         ),
-        "_detekt_wrapper_jar": attr.label(
-            default = "//detekt/wrapper:bin_deploy.jar",
-            allow_single_file = True,
+        "_detekt_wrapper": attr.label(
+            default = "//detekt/wrapper:bin",
```

Comment:
Hah, well, that’s ok but I’m wondering what actual jvm do we end up running here

There is one coming with Bazel, there is one coming from rules_java, you should be able to check the runfiles Bazel generates or that bin tartget’s output is a script itself

Reply:
Sorry I’m on ipad I’d check myself

Reply:
```
bin.runfiles/remotejdk11_macos/bin/java
```

Seems like the `rules_java` one.